neurom/io/tests/test_utils.py

```diff
@@ -92,21 +93,53 @@ def _check_neurites_have_no_parent(nrn):
         nt.assert_true(n.root_node.parent is None)
 
 
+def test_get_morph_files():
+    ref = set(['Neuron_h5v2.h5', 'Neuron_2_branch_h5v2.h5', 'Neuron_slice.h5',
+               'Neuron.swc', 'Neuron_h5v1.h5', 'Neuron_2_branch_h5v1.h5'])
+
+    FILE_PATH = os.path.abspath(os.path.join(DATA_PATH, 'valid_set'))
+    files = set(os.path.basename(f) for f in utils.get_morph_files(FILE_PATH))
+
+    nt.assert_equal(ref, files)
+
+
 def test_load_neurons():
+    # List of strings
     nrns = utils.load_neurons(FILES, neuron_loader=_mock_load_neuron)
     for i, nrn in enumerate(nrns):
         nt.assert_equal(nrn.name, _get_name(FILES[i]))
     nt.assert_raises(SomaError, utils.load_neurons, NO_SOMA_FILE)
 
+    # Single string
+    nrns = utils.load_neurons(FILES[0], neuron_loader=_mock_load_neuron)
+    nt.assert_equal(nrns[0].name, _get_name(FILES[0]))
 
-def test_get_morph_files():
-    ref = set(['Neuron_h5v2.h5', 'Neuron_2_branch_h5v2.h5', 'Neuron_slice.h5',
-               'Neuron.swc', 'Neuron_h5v1.h5', 'Neuron_2_branch_h5v1.h5'])
+    # Single Path
+    nrns = utils.load_neurons(Path(FILES[0]), neuron_loader=_mock_load_neuron)
+    nt.assert_equal(nrns[0].name, _get_name(FILES[0]))
 
-    FILE_PATH = os.path.abspath(os.path.join(DATA_PATH, 'valid_set'))
-    files = set(os.path.basename(f) for f in utils.get_morph_files(FILE_PATH))
+    # sequence of strings
+    iter_file = (f for f in FILES)
+    nrns = utils.load_neurons(iter_file, neuron_loader=_mock_load_neuron)
+    for i, nrn in enumerate(nrns):
+        nt.assert_equal(nrn.name, _get_name(FILES[i]))
+    nt.assert_raises(SomaError, utils.load_neurons, NO_SOMA_FILE)
```

Comment:
Why do we keep repeating this line below? Should not be one time enough?

Reply:
Because I'm dumb XD